pysat/tests/test_orbits.py

```diff
@@ -405,160 +434,38 @@ def filter_data(inst):
-        self.testInst.orbits.prev()
-        control = self.testInst.copy()
-        for j in range(10):
-            self.testInst.orbits.next()
-        for j in range(10):
-            self.testInst.orbits.prev()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_symmetric_single_day_0_UT(self):
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
-        control = self.testInst.copy()
-        for j in range(10):
-            self.testInst.orbits.next()
-        for j in range(10):
-            self.testInst.orbits.prev()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_symmetric_multi_day_0_UT(self):
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
-        control = self.testInst.copy()
-        for j in range(20):
-            self.testInst.orbits.next()
-        for j in range(20):
-            self.testInst.orbits.prev()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_symmetric_single_day_off_0_UT(self):
-        """ Test successful orbit calls for a day about a time off 00:00 UT
-        """
-        self.stime -= dt.timedelta(days=1)
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
-        control = self.testInst.copy()
-        for j in range(10):
-            self.testInst.orbits.next()
-        for j in range(10):
-            self.testInst.orbits.prev()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_symmetric_multi_day_off_0_UT(self):
-        """ Test successful orbit calls for days about a time off 00:00 UT
-        """
-        self.stime -= dt.timedelta(days=1)
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
-        control = self.testInst.copy()
-        for j in range(20):
-            self.testInst.orbits.next()
-        for j in range(20):
-            self.testInst.orbits.prev()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_antisymmetric_multi_day_off_0_UT(self):
-        """ Test successful orbit calls for different days about a time off 0 UT
-        """
-        self.stime -= dt.timedelta(days=1)
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
-        control = self.testInst.copy()
-        for j in range(10):
-            self.testInst.orbits.next()
-        for j in range(20):
-            self.testInst.orbits.prev()
-        for j in range(10):
-            self.testInst.orbits.next()
-        assert all(control.data == self.testInst.data)
-
-    def test_repeated_orbit_calls_antisymmetric_multi_multi_day_off_0_UT(self):
-        """ Test successful orbit calls for more days about a time off 0 UT
-        """
-        self.stime -= dt.timedelta(days=1)
-        self.testInst.load(date=self.stime)
-        self.testInst.orbits.next()
```

Comment:
This test (and similar) can be combined with an extra level of parametrize, but I'm not sure it adds anything new